monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -175,30 +175,42 @@ groups:
           description: >
             Root volume (OSD and MON store) is dangerously full: {{ $value | humanize }}% free.
 
-      # alert on nic packet errors and drops rates > 1 packet/s
+      # alert on nic packet errors and drops rates > 1% packets/s
```

Comment:
Is this metric based on a customer case requirement? Shouldn't we put a more strict rate? 1% PER would be something more expected from a wireless network. On a 10 GbE link without jumbo frames at max theoretical throughput, a 1% PER means more than 8,000 lost/dropped packets per second, and we should probably raise a flag far before that happens.

Something between the former "1 packet/s" threshold and "8,000 packets/s".

Reply:
@p-se, ping?

Reply:
@epuertat Yes, it's a requirement from a customer. A percentage seems to cover a failure rate quite adequately. 0.5% would be in the middle of the requested range between "1 packet/s" and "8,000 packet/s" (on a 10GbE link without jumbo frames).

Would that be suitable or do you have a different opinion?

Reply:
My main concern with this one is that, with such a high value (8,000, 4,000 pkts/s or 2,000 pkts/s are still high ones), we are increasing the threshold so much that the alert will become useless most of the time. Packets lost/dropped is usually an indication of a congested network or a saturated system (CPUs not able to cope with the incoming traffic).

Don't we have a clear understanding about what's going on with that deployment and what's the acceptable threshold for them? If this is just a unique customer issue, and not the general trend, why not simply silencing this alert?

I'd be happy if we could have some feedback from field/customer folks: @pcuzner, @mhackett, @jan--f, @wido. What do you think it would a valid threshold here?

Reply:
Thanks a lot @wido for your feedback!

> Don't we have a clear understanding about what's going on with that deployment and what's the acceptable threshold for them? If this is just a unique customer issue, and not the general trend, why not simply silencing this alert?

@p-se what do you think ?

Reply:
I think none of us have hard data as to what the best warning threshold is; clearly it should alert *before* it turns problematic. It's a heuristic only for admins to look at their networking/load and go "yeah that checks out" or "uh-oh".

I think we should go with something like `min(0.01%/s or 10/s) over a minute`.

We just don't want to immediately trigger because of a minor packet drop. They may have to adjust it for their environment anyway if that's an expected normal occurrence because they under-spec'ed the fabric.

Reply:
Thanks, @l-mb ! We nevertheless agree on decreasing the threshold, so ok then let's add some guard time to decrease sensitivity to bursts. @p-se are you ok with the change proposed by @l-mb?

Reply:
>  @p-se are you ok with the change proposed by @l-mb?

Yes. I'll take care of the adaptation.

Note that the period evaluated for the current implementation is five minutes and for the proposed implementation is one minute, so we should have some guard time already. I'd like to stick to one minute, if you don't have any objections.

Reply:
> Note that the period evaluated for the current implementation is five minutes and for the proposed implementation is one minute, so we should have some guard time already. I'd like to stick to one minute, if you don't have any objections.

@p-se, it makes fully sense to me. 5 mins it's probably too long. Thanks!